src/styles/group-list.scss

```diff
@@ -87,6 +87,11 @@ $group-list-spacing-below: 50px;
   transform: translateY(1px);
 }
 
+.group-list-label__icon--third-party {
+  height: 15px;
+  width: 15px;
```

Comment:
The size looks good. Assuming that the icon uses the available height, this results in the bottom of the icon being aligned with the baseline of the group name to the right and the top of the icon being a couple of px above it. Moving it down a couple of px using relative positioning makes it look more balanced IMO. Thoughts?